sycl/include/CL/sycl/handler.hpp

```diff
@@ -386,7 +386,19 @@ class __SYCL_EXPORT handler {
                        static_cast<int>(AccessTarget), ArgIndex);
   }
 
-  template <typename T> void setArgHelper(int ArgIndex, T &&Arg) {
+  template <typename T> struct ShouldEnableSetArgHelper {
+    static constexpr bool value = std::is_trivially_copyable<T>::value
+#ifdef CL_SYCL_LANGUAGE_VERSION
+#if CL_SYCL_LANGUAGE_VERSION <= 121
```

Comment:
May it be rewritten as:
`#if CL_SYCL_LANGUAGE_VERSION && CL_SYCL_LANGUAGE_VERSION <= 121`

Reply:
Applied. Thanks for suggestion!